src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -24,23 +25,40 @@ private void Render(List<CodeLine> list, IEnumerable<CodeFileToken> node)
             string currentId = null;
             bool isDocumentation = false;
             bool isDeprecatedToken = false;
+            //This will be set to true by default when a new line starts and 
+            // set to false when any non documentation token is found within the line
+            bool isLineAllDocumentation = true;
```

Comment:
What about empty lines?

Reply:
correct. I should set this as false and set it true when range started and if line is empty. Similarly this flag should not be blindly reset to tru when a new line starts and it should be set if it is only within range. Thanks for the suggestions.